docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -436,8 +423,12 @@ testing PRL algorithms, so we will keep it quite simple.
 
 These notes on ACS data sources on migration could be useful for the
 more complex rates in the future.  For now, let's make both households
-and simulants move at a rate of 15 moves per 100 person-years
-(independently).
+and simulants (who are not the reference person) move at a rate of 15
```

Comment:
see edits, which are intended to be "simple"... let me know if they are not.

Reply:
i think i didn't ask my question clearly enough -- this might be obvious but i wasn't sure because i could imagine a few ways to interpret this:

on a timestep for a simulant, to get probability they will move, it will be something like 0.15 * (1/(time steps per year))

should the probability a household moves be equal to this (i.e., 15 moves per household-year), as opposed to something like 15 moves per total person-years lived by the household? which i think would be something like

0.15 * number_of_household_members * (1/(time steps per year))

or 15 moves per average person-year per person in the household? (which, if people and households can each only move on timesteps, might be equal to counting household years)